### PR TITLE
doc: add details about running tests locally in CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by kjin**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,3 +1,50 @@
...
+`npm run script` is an alias for `ts-node -p ./scripts ./scripts/index.ts`.
+
+For example, to compile all scripts and then initialize test fixtures, the command to use would be one of:
+
+```bash
+# Option 1
+ts-node -p ./scripts ./scripts/index.ts npm-compile-all init-test-fixtures
+
+# Option 2
+npm run script npm-compile-all init-test-fixtures
```
Comment (jinwoo) [inline CONTRIBUTING.md]: Don't you need `--` between `script` and the rest of the arguments?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,3 +1,50 @@
...
+
+# Option 3
+npm run compile-all
+npm run script init-test-fixtures
```
Comment (jinwoo) [inline CONTRIBUTING.md]: Same question here.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,3 +1,50 @@
...
+npm run compile
+npm run init-test-fixtures
+npm test # Or "npm run script run-unit-tests"
+```
+
+A convenient one-liner for this (after `npm install`) is:
+
+```bash
+npm install
+GCLOUD_TRACE_NEW_CONTEXT=1 npm run script npm-compile-all init-test-fixtures run-unit-tests
```
Comment (jinwoo) [inline CONTRIBUTING.md]: And here.

File: CONTRIBUTING.md (hunk 1 above)
Comment (kjin) [reply CONTRIBUTING.md]: We don't need it. I believe `npm run x` is mapped to its underlying script first, and then the rest of the arguments are passed to the result of that evaluation.

Comment (jinwoo) [reply CONTRIBUTING.md]: OK. Maybe I was misunderstanding something.

**Pull request merged**